docs/source/internals.rst

```diff
@@ -0,0 +1,111 @@
+.. module:: jupyter_contrib_nbextensions.nbconvert_support
+
+Notebook extension structure
+============================
+
+The nbextensions are stored each as a separate subdirectory of
+`src/jupyter_contrib_nbextensions/nbextensions`
```

Comment:
it seems the single backticks [give the default role](http://docutils.sourceforge.net/docs/ref/rst/roles.html), which [is configurable](http://docutils.sourceforge.net/docs/ref/rst/directives.html#default-role). Perhaps we should add a

```python
default_role = 'code'
```

to `conf.py`?